package.json

```diff
@@ -59,9 +62,11 @@
     "gulp-uglify": "1.5.3",
     "gulp-util": "3.0.7",
     "gulp-webserver": "0.9.1",
+    "moment": "2.18.1",
     "ngify": "1.4.1",
     "npm-shrinkwrap": "200.5.1",
     "react-addons-test-utils": "0.14.8",
+    "react-time": "4.3.0",
```

Comment:
Yep, missed that one. Both `react-time` and `moment` are supposed to be dependencies, not devDependencies. Will change it in my PR.